### PR TITLE
GEOMESA-1222 Docs: fix CSS styling issues

### PR DESCRIPTION
**Pull request by mdzimmerman**
* Change background of admonition boxes so they are more readable
* Make highlighted text sections off-white
* Fix colors of table rows

Signed-off-by: Matthew Zimmerman <matt.zimmerman@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
Nice, looks a lot better. I might push up some more tweaks to a few things that look a little glaring IMO, @mdzimmerman mind giving feedback later?

**Comment by mdzimmerman** [comment]
I'd be happy to.

**Comment by elahrvivaz** [comment]
@mdzimmerman updated some things - take a look if you don't mind.

File: docs/_static/css/theme_custom.css [hunk 1]
```diff
@@ -96,35 +96,49 @@ input[type="file"]:focus:invalid:focus,input[type="radio"]:focus:invalid:focus,i
...
 .wy-menu-vertical a:active {
-   background-color: #E6A150;
+  background-color: #E6A150;
 }
 
 .wy-side-nav-search {
-   background-color: #E6A150;
-   color: #333;
+   background-color: #333333;
+   color: #e6a150;
```
Comment (mdzimmerman) [inline docs/_static/css/theme_custom.css]: I kinda prefer an orange background for the box in the upper left hand corner; it's a nice contrast to the other sections of the document.

Comment (elahrvivaz) [inline docs/_static/css/theme_custom.css]: hehe, i think it looks really glaring... @jnh5y @jahhulbert-ccri @cne1x opinions?

File: docs/_static/css/theme_custom.css [hunk 2]
```diff
@@ -96,35 +96,49 @@ input[type="file"]:focus:invalid:focus,input[type="radio"]:focus:invalid:focus,i
...
    color: #333;
 }
 
 .wy-side-nav-search>div.version {
-   color: #666;
+   color: #bbb;
+}
+
+.wy-side-nav-search input[type=text] {
+    background-color: #e3e3e3;
```
Comment (mdzimmerman) [inline docs/_static/css/theme_custom.css]: Yeah, this is a good catch. I tried to get rid of FFF backgrounds everywhere else but missed this one.

File: docs/index.rst [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 .. toctree::
     :maxdepth: 1
 
-    user/index
-    developer/index
-    tutorials/index
+    User Manual <user/index>
+    Developer Guide <developer/index>
+    Tutorials <tutorials/index>
```
Comment (mdzimmerman) [inline docs/index.rst]: [picky] Is this necessary if you've retitled the child pages?

Comment (elahrvivaz) [inline docs/index.rst]: Yeah, good point - i did this initially then decided to rename the pages themselves.

File: docs/_static/css/theme_custom.css (hunk 1 above)
Comment (mdzimmerman) [inline docs/_static/css/theme_custom.css]: old: www.geomesa.org/documentation/
new: www.geomesa.org/documentation/test/

**Comment by elahrvivaz** [comment]
I really wanted to change the 'Docs' in the breadcrumbs to 'GeoMesa', but it appears to be hard-coded in the theme.

File: docs/_static/css/theme_custom.css (hunk 1 above)
Comment (mdzimmerman) [inline docs/_static/css/theme_custom.css]: If I get outvoted on the orange vs. gray thing, make sure you change the background color of ``.wy-nav-top`` too, which is only rendered when the view is narrowed to mobile device widths.

Comment (elahrvivaz) [inline docs/_static/css/theme_custom.css]: I like keeping that orange, actually. It works a lot better as a discrete element across the top vs just on the top-left.

**Comment by elahrvivaz** [comment]
@mdzimmerman can you update the test deployment?

**Comment by mdzimmerman** [comment]
http://geomesa.org/documentation/test/ is updated.

**Pull request merged**